### PR TITLE
Unfork import statements

### PR DESCRIPTION
**Pull request by stvstnfrd**
Some of these were duplicates, others unused entirely. The rest aim to
minimize the diff by moving our additions to a new line.

By having our delta as a single addition instead of both a deletion and
addition, we reduce the change of future conflict.

This is just the start; there will be more of these to follow in the
future.

### CODE REVIEW

**Comment by stvstnfrd** [comment]
At least one of @caesar2164 @kluo @dcadams please review.

**Comment by caesar2164** [comment]
do we not need the mock imports?

File: cms/djangoapps/contentstore/tests/test_users_default_role.py [hunk 1]
```diff
@@ -2,7 +2,6 @@
 Unit tests for checking default forum role "Student" of a user when he creates a course or
 after deleting it creates same course again
 """
-import mock
```
Comment (stvstnfrd) [inline cms/djangoapps/contentstore/tests/test_users_default_role.py]: Not used in this file.

Comment (stvstnfrd) [inline cms/djangoapps/contentstore/tests/test_users_default_role.py]: Wait, maybe I'm looking in the wrong place..

Comment (stvstnfrd) [inline cms/djangoapps/contentstore/tests/test_users_default_role.py]: Odd, not sure where that came from.. could have sworn I hit an "unused import" violation.. Will add back.

File: cms/djangoapps/contentstore/tests/test_permissions.py [hunk 2]
```diff
@@ -2,7 +2,6 @@
 Test CRUD for authorization.
 """
 import copy
-import mock
```
Comment (stvstnfrd) [inline cms/djangoapps/contentstore/tests/test_permissions.py]: Odd, not sure where that came from.. could have sworn I hit an "unused import" violation.. Will add back.

**Comment by caesar2164** [comment]
looks good to me, :+1:

**Pull request merged**